SetupScript.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+	then
+	echo "Permission Denied"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c3 google.com > /dev/null
+
+if [ $? -ne 0 ]
+then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+cd /opt/
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git
+cd Epsilon-Raspberry/primary/
+cp xorg.conf /etc/X11/
+cp config.txt /boot/
+
+#install rabbit-mq
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | tee /etc/apt/sources.list.d/rabbitmq.list && apt-get update && apt-get install rabbitmq-server
+apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev software-properties-common build-essential mesa-common-dev libgl1-mesa-dev
+
+cd /tmp/
+if ls /usr/local/lib/librabbitmq.* 1> /dev/null 2>&1 ;
+then
+echo "Rabbitmq already setup"
+else
+git clone https://github.com/alanxz/rabbitmq-c
+mkdir rabbitmq-c/build && cd rabbitmq-c/build
+cmake ..
+cmake --build .
+cp librabbitmq/*.a /usr/local/lib/
+cp librabbitmq/*.so* /usr/local/lib/
+fi
+
+#begin QT install
+apt-get update
+apt-get --yes upgrade
+apt-get install --yes libfontconfig1-dev libdbus-1-dev libfreetype6-dev libudev-dev libicu-dev libsqlite3-dev
```

Comment:
This is hard to read. I know it'll be pretty long but I think this format would be better:

```bash
apt-get install --yes \ 
    libfontconfig1-dev \
    libdbus-1-dev \
    libudev-dev \
    ...
```

I'd also recommend the quiet flag, `-qq`